lib/spack/docs/config_yaml.rst

```diff
@@ -184,3 +184,12 @@ other tools may need custom ``LD_LIBRARY_PATH`` setings to run.  You can
 set ``dirty`` to ``true`` to skip the cleaning step and make all builds
 "dirty" by default.  Be aware that this will reduce the reproducibility
 of builds.
+
+--------------------
+``ccache``
+--------------------
+
+When set to ``true`` Spack will use ccache to cache compiles. This is
+useful specifically un two cases: (1) Use with ``spack setup``, (2)
+Build the same package with many different variants. The default is
+``false``.
```

Comment:
Could more documentation be useful?  For example... How does Spack find ccache?  Are you supposed to ask Spack to build it first?  How should ccache be configured?